packages/react-components/react-list-preview/src/components/ListItem/useListItem.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { useFocusableGroup } from '@fluentui/react-tabster';
+import { getIntrinsicElementProps, slot, useEventCallback, useId } from '@fluentui/react-utilities';
+import { Space } from '@fluentui/keyboard-keys';
+import type { ListItemProps, ListItemState } from './ListItem.types';
+import { useListContext_unstable } from '../List/listContext';
+import { Checkmark16Filled } from '@fluentui/react-icons';
+
+const DEFAULT_ROOT_EL_TYPE = 'li';
+
+const listPropsForSelected = {
+  tabIndex: 0,
+  role: 'option',
+  'aria-selected': true,
+  checkmark: {
+    children: <Checkmark16Filled />,
+  },
+};
+
+const listPropsForNotSelected = {
+  tabIndex: 0,
+  role: 'option',
+  'aria-selected': false,
+  checkmark: {
+    children: null,
+  },
+};
```

Comment:
```suggestion
const listPropsForSelected = {
  tabIndex: 0,
  role: 'option',
  'aria-selected': true,
};

const listPropsForNotSelected = {
  tabIndex: 0,
  role: 'option',
  'aria-selected': false,
};
```

As I see `checkbox` is handled by `slot.optional(props.checkmark)` and can be removed. As the single difference then is `'aria-selected'` - it's questionable why `listPropsForSelected` & `listPropsForNotSelected` are needed then ;)

Reply:
good catch, removing this